src/dom_manipulation.js

```diff
@@ -0,0 +1,18 @@
+// variables
+const scoreTable = document.querySelector('#score_table');
+// functions
+const createScore = (scoreData) => {
+  // create elements
+  const tableRow = document.createElement('tr');
+  const tableName = document.createElement('td');
+  const tableScore = document.createElement('td');
+  // set properties
+  tableName.textContent = scoreData.user;
+  tableScore.textContent = scoreData.score;
+  // append
+  tableRow.appendChild(tableName);
+  tableRow.appendChild(tableScore);
+  scoreTable.appendChild(tableRow);
+};
+
+module.exports = { createScore };
```

Comment:
- [OPTIONAL] It would be better to take advantage of ES6 in this line. ES6 conventions recommend that we do default exports when a module has one function to be exported. Consider refactoring as below.
```suggestion
export default createScore;
```

Reply:
Once again, I appreciate your feedback.
I'll check that to make the right implementation of modules.